README.md

```diff
@@ -2,7 +2,10 @@
 
 ## Preface
 
-A lot of information has been written about JavaScript and EcmaScript since both the language creation and the standard definition. The intention of this road map is to share with you one of many path you can take to grok the language and specially to point you to the high quality information in the intent to save you from hitting misleading ( when not completely wrong ) documents outside in the wild. Hopefully your walk is much easier than mine!
+A lot of information has been written about JavaScript and EcmaScript since both the language creation time and the standard definition time.
+The intention of this road map is to share with you one of many path you can take to grok the language and specially to point you to the high quality information  (which I won't be ever able to write in a better way) in the intent to save you from hitting misleading ( when not completely wrong ) documents outside in the wild.
```

Comment:
Typo in word 'grok', should be 'grow':
"...many path you can take to **grok** the language and specially..."

Aside from being misspelled, I don't think the phrasing is correct. You can't grow a language.

Reply:
GROK is a beautiful nerdish neologism I love to use
https://en.wikipedia.org/wiki/Grok

embrace it, you'll love it once lo grok it ;)